bin/postInstall

```diff
@@ -14,11 +14,6 @@ if (["darwin", "linux"].includes(process.platform)) {
   run('pkill -f "cli.js start" || set exit 0')
 }
 
-// On iOS, make sure CocoaPods are installed
-if (process.platform === "darwin") {
-  run('if [ -d "ios" ]; then cd ios && pod install && cd -; fi')
-}
-
```

Comment:
this is no longer needed, we'll have expo manage that folder. whenever a package is added, make sure to run `expo prebuild --clean`.